### PR TITLE
[ST-446] Add AvatararOrGravatar and GravatarImageUrl

### PR DESCRIPTION
**Pull request by iezer**
Will allow us to remove those components from es-student, es-certification, and profile-engine

Related PRs
==========

https://github.com/PrecisionNutrition/es-certification/pull/16
https://github.com/PrecisionNutrition/profile-engine/pull/1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,10 +41,12 @@
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^2.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-md5": "^1.2.0",
```
Comment (jherdman) [inline package.json]: Wouldn't this be a `"dependency"`?

Frankly I still find the dependency differences baffling with respect to Ember.

Comment (iezer) [reply package.json]: Yes good call. I just got that issue when integrating with `profile-engine`. Let's keep this PR open until I've tested in the apps.

**Comment by iezer** [comment]
@jherdman This is good to merge. Tests on other PRs are passing. Please merge this one and then I'll update the versions in the other repos.

https://github.com/PrecisionNutrition/profile-engine/pull/1
https://github.com/PrecisionNutrition/es-certification/pull/16

**Comment by jherdman** [comment]
@iezer merged.

**Pull request merged**